stdlib/public/core/Array.swift

```diff
@@ -1290,6 +1290,25 @@ extension Array: RangeReplaceableCollection {
   }
 }
 
+// Implementations of + and += for same-type arrays. This combined
+// with the operator declarations for these operators designating this
+// type as a place to prefer this operator help the expression type
+// checker speed up cases where there is a large number of uses of the
+// operator in the same expression.
+extension Array {
+  @inlinable // FIXME(sil-serialize-all)
```

Comment:
We don't need the FIXMEs here. These are legitimately inlinable.

Reply:
Okay I'll remove those in a separate PR. This was copy/pasted from elsewhere.